deployment/united-manufacturing-hub/values.yaml

```diff
@@ -2198,6 +2202,24 @@ packmlmqttsimulator:
       cpu: "100m"
       memory: "100Mi"
 
+### tulip-connector ###
+tulipconnector:
+  image:
+    repository: unitedmanufacturinghub/tulip-connector
+    pullPolicy: IfNotPresent
+    # Only specify tag if you want to use a specific version. If not specified the latest stable version is automatically selected
+    # tag: main
+  replicas: 1
+  env:
+    mode: prod
```

Comment:
can you add documentation on how to use it? and an option to enable/disable it? it should be disabled by default

Reply:
The option to enable/disable is under _000_commonConfig.tulipconnector.enabled and is set to false.
The values you are quoting shouldn't be changed unless someone knows what he's doing.
All of this is written in the documentation, but I'm waiting to publish it until I have written the full API specification